src/ConsoleApp/Program.cs

```diff
@@ -4,6 +4,7 @@
 //WeatherForecast weatherForecast = new WeatherForecast(new HttpClient());
 
 var job = DependencyInjection.Resolve<MainJob>();
+var mediator = DependencyInjection.Resolve<MainJob>();
```

Comment:
Remove the line.

Reply:
U haven't removed it

Reply:
Sorry, thought you asked me to remove comment. Done!!!